app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 heals: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = heals
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+
+class Carnivore(Animal):
+    @classmethod
+    def bite(cls, herbivore: Animal) -> None:
```

Comment:
Code Efficiency: You should use the class name to get access to the class attributes. In this case, instead of `cls.alive.remove(herbivore)`, you should use `Animal.alive.remove(herbivore)`. Refer to point 1 in the checklist.